osv/ecosystems/_ecosystems.py

```diff
@@ -64,7 +64,7 @@
     'Linux': OrderingUnsupportedEcosystem(),
     'OSS-Fuzz': OrderingUnsupportedEcosystem(),
     'Photon OS': OrderingUnsupportedEcosystem(),
-    'Red Hat': OrderingUnsupportedEcosystem,
+    'Red Hat': OrderingUnsupportedEcosystem(),
```

Comment:
We should create a proper RedHat ecosystem in a followup PR, and we should have RPM ordering already so API queries will work out of the box.